services/order.py

```diff
@@ -0,0 +1,33 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from db.models import Order, Ticket
+
+
+def create_order(tickets, username, date=None):
+    with transaction.atomic():
+        user = get_user_model().objects.get(username=username)
+        order = Order.objects.create(user=user)
+
+        if date:
+            order.created_at = date
+
+        order.save()
```

Comment:
You must save after setting another date in the `if date:` condition.

Reply:
thanks